post.js

```diff
@@ -106,8 +106,18 @@ class DistributedPost extends HTMLElement {
     postContent.classList.add('post-content')
 
     // Determine content source based on structure of jsonLdData
-    const contentSource =
-      jsonLdData.content || (jsonLdData.object && jsonLdData.object.content)
+    let contentSource = jsonLdData.content || (jsonLdData.object && jsonLdData.object.content)
+    // Check if the content has a mention and update the href attribute
+    if (jsonLdData.tag && jsonLdData.tag.some(tag => tag.type === 'Mention')) {
```

Comment:
```suggestion
if(jsonLdData.tag) {
  const mentions = jsonLdData.tag.filter(({type}) => type === 'Mention')
  if(mentions.length) {
```
It should be cheaper to precompute the list of mentions instead of going through all the tags twice. Also instead of forEach could you stick to `for of`?